<launch>
   <arg name="drone_id" default="drone1"/>
   
   <group ns="$(arg drone_id)">
   	<node pkg="bridge_px4" type="setpoint_publisher" name ="setpoint_publisher" output="screen">
	      <param name="sp_out_hz" value="100.0"/>
         <param name="checkup_hz" value="10.0"/>
         <param name="sp_gcs_hz_min" value="2.0"/>
         <param name="checkup_hz_min" value="5.0"/>
      </node>
   </group>
</launch>
